### ISSUE TITLE
Accessibility review for org and legacy/org switching

### ISSUE THREAD

**Issue by katypies** [cisagov/manage.get.gov#4270]
### Sub-issue description

For UAT on the switcher page, and switching drop-down, we don't currently have any explicit accessibility testing, but we should incorporate that.

### Acceptance criteria

- [x] Meet with Kaitlin to get an overview of the UAT materials and what's needed.
- [x] Review [existing test scripts](https://docs.google.com/document/d/17qGbyY2AUpmjU16UH5CW8wKay_WC7A2NFKMKSR0vDJI/edit?tab=t.gvjferiogogp) to determine if accessibility checks should be built into them, or executed separately
- [x] Identify key accessibility tests that should be conducted for these two switching models, and add to the UAT document
